cloud/services/shared/tags.go

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package shared
```

Comment:
nit: i don't think we specifically need this in shared, it could just be a `tags` packageinstead

Reply:
But lets leav it here for the time being. So no action needed on this comment.

Reply:
My reason to put this in `shared` is that resource manager tags are used for different types of resources (compute, disks, clusters, etc.) and I didn't think of tags as resources at the level of other `cloud/services` like compute or container. Hopefully this makes sense.

Reply:
Good point, thanks @salasberryfin